Tests/Core/AWSClientRuntimeTests/AWSClientConfigurationTests.swift

```diff
@@ -1,82 +0,0 @@
-//
-// Copyright Amazon.com Inc. or its affiliates.
-// All Rights Reserved.
-//
-// SPDX-License-Identifier: Apache-2.0
-//
-
-import XCTest
-import AWSClientRuntime
-
-class AWSClientConfigurationTests: XCTestCase {
```

Comment:
Was this meant to be deleted?

Reply:
Yes, the `AWSClientConfiguration` was deleted along with the new Config changes from Andrew.